release-notes/2023-10/README.md

```diff
@@ -0,0 +1,252 @@
+<!-- This file should be renamed to README.md and placed in the directory for the release. -->
+
+# SystemLink Enterprise release 2023-10 Release Notes
+
+The 2023-10 release bundle for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## Upgrading from the release 2023-09 to the release 2023-10
+
+<!-- Optional section to include comments and instructions needed to successfully upgrade from the previous release to the current release. If the only changes needed are already captured in Helm Chart Breaking Changes, this section is not needed. -->
+
+## New Features and Behavior changes
```

Comment:
This section should be a summary of the changes and links to the documents or configuration (if available/necessary). See past release notes for examples. 

The upgrade instructions that are detailed here should be in the **Upgrade Considerartions** section. If the instructions are long we should consider having them in their own document and referencing them.

Reply:
@prestwick ,

I kept this under New Features and Behavior Changes:
- testinsightsui 0.6.112: Comments service was added to top level helm chart in previous release 2023-09. In this release, feature flag is removed and comments tab will be visible in result details page and user can add/view/edit/delete comments under a test result.
- The Test Analytics privilege category has been added, and includes the Query Measurements privilege. This privilege is not currently functional and is being added in support of features that will release in a future version.

I moved these to section `Upgrade from the release 2023-09 to the release 2023-10':
- webserver 0.13.4: Upgraded Redis dependency from 7.0 to 7.2. This is a breaking change. It is necessary to upgrade the entire cluster in parallel, which is not something Kubernetes will do automatically.

I moved these to section 'Upgrade Considerations':
- dataframeservice 0.14.49: The Dremio data set refresh job interval was increased from 2 minutes to 1 hour....Customers are not required to uptake this change, but doing so will...
- dataframeservice 0.14.49: The DataFrame Service now uses a more efficient method for writing data to new tables, replacing Kafka...
AND I'm going to put this long info as a separate .md file.

Reply:
@prestwick , I've additionally moved the steps for Kafka removal to this .md file:
- https://github.com/ni/install-systemlink-enterprise/blob/sle-release-notes-2023-10/release-notes/2023-10/Remove-Kafka-from-the-cluster.md

Setting this as Resolved.